integration/js/README.md

```diff
@@ -142,12 +142,31 @@ Run the test:
-    On sauce labs virtual machines files are stored in the below folder locations
-    Windows = C:\Users\Administrator\Downloads
-    Mac = /Users/chef/Downloads
-    Linux = /home/chef/Downloads
-- Upload the pre-run executable to Sauce Labs storage by using Sauce Labs API methods: https://docs.saucelabs.com/dev/api/storage/
-- Add the pre-run to Sauce configs in your code like this prerun: 'storage:filename=filename.ext'. You can also use storage id of the file location. 
+## Testing against audio/video stream in Saucelabs
+Some tests might require testing against a custom audio/video stream. To test against a custom audio or video stream, you need the media file on Sauce Labs virtual machine before running your test. For that you can use a pre-run executable script to download file from a public storage to the virtual machine before running your tests. A pre-run executable script is a minimal `bash` script that will use `curl` to install the resources on the Sauce Labs VM. Pre-run executable scripts can be used for several different use cases like changing VM configurations and browser settings. In our use case, we are using the pre-run executable to download audio/video assets. More information on pre-run scripts can be found [here](https://docs.saucelabs.com/web-apps/automated-testing/selenium/pre-run-executables/).
+
+1. Create a pre-run script by following the steps given in [downloading files to a vm prior to testing](https://docs.saucelabs.com/web-apps/automated-testing/selenium/pre-run-executables/#downloading-files-to-a-vm-prior-to-testing). Following is an exmaple of a pre-run script that will download public assets and store them at the specified path.
+```shell
+#!/bin/bash
+curl -o /Users/chef/file.txt http://mywebsite.com/file.txt
+```
+
+2. Once the script is created, you should then upload the pre-run executable to Sauce Labs storage by using Sauce Labs API methods: [Saucelabs Storage CRUD API](https://docs.saucelabs.com/dev/api/storage/). You will need your Saucelabs account username and access key for authorization. Upon successful upload you will get a storage id in the HTTP response. This id will be useful in step 4 for setting the config.
+
+3. In your test config, configure the directory where the audio/video assests will be downloaded to.
+
+    - Windows = `C:\Users\Administrator\`
+
+    - Mac = `/Users/chef/`
+
+    - Linux = `/home/chef/`
+
+> NOTE: The path specified in the test config should match the path specified in the `-o` option of the curl command in step 2.
+
+4. Add the pre-run script to Sauce configs in your code like this prerun: 'storage:filename=filename.ext'. You can also use the name of the pre-run script file but id is the better option. For reference,
+    ```js
+        // integration/js/utils/WebdriverSauceLabs.js
+        const getPrerunScript = (capabilities) =>{
+            return capabilities.name.includes("Background Blur Test") ? 'storage:cf5a00cc-2a0d-40bb-9f2e-f721f2eec1f4' : "";
```

Comment:
Is this an actual storageId? If so please remove it? We probably also don't want to expose our storage Id in the test either.

Reply:
This is not a valid storage id. Also, the storage id is already publically exposed in the code.

Reply:
Can we just change it to storage:<storage session id> to avoid confusion

Reply:
Done